tests/common/dualtor/dual_tor_utils.py

```diff
@@ -196,6 +198,40 @@ def force_active_tor(dut, intf):
     dut.shell_cmds(cmds=cmds)
 
 
+def force_active_tor_swssconfig(duthost, intf):
```

Comment:
Please use the fixtures defined here to configure the ToR as active/standby for consistency: https://github.com/Azure/sonic-mgmt/pull/2945

Reply:
Got it, will update accordingly.

Reply:
Could you please take a review again @theasianpianist ?